scripts/lib/settings-window-capture/cdp.mjs

```
@@ -0,0 +1,209 @@
+  });
+
+  return {
+    /**
+     * Send one command and await its result. `sessionId` targets a page
+     * session rather than the browser itself.
+     */
+    send(method, params = {}, sessionId) {
+      const id = nextId++;
+
+      return new Promise((resolve, reject) => {
+        const timer = setTimeout(() => {
```

ESLint

[error] 'setTimeout' is not defined.

(no-undef)